<?xml version='1.0' encoding='utf-8'?>
<robot name="robot" xmlns:xacro="http://www.ros.org/wiki/xacro">

  <xacro:property name="prefix" value="robot_" />
  <xacro:property name="front_axle_steering_joint" value="joint1" />
  <xacro:property name="front_left_wheel_steering_joint" value="joint2" />
  <xacro:property name="front_right_wheel_steering_joint" value="joint3" />
  <xacro:property name="front_left_wheel_spinning_joint" value="joint4" />
  <xacro:property name="front_right_wheel_spinning_joint" value="joint5" />
  <xacro:property name="rear_left_wheel_spinning_joint" value="joint6" />
  <xacro:property name="rear_right_wheel_spinning_joint" value="joint7" />

  <xacro:property name="simulation_hardware_plugin" value="foo" />
  <xacro:property name="wheelbase" value="1.4" />
  <xacro:property name="front_track" value="1.2" />
  <xacro:property name="front_wheel_radius" value="0.4" />
  <xacro:property name="front_hub_carrier_offset" value="0.1" />
  <xacro:property name="rear_track" value="1.4" />
  <xacro:property name="rear_wheel_radius" value="0.6" />
  <xacro:property name="rear_hub_carrier_offset" value="0.15" />
  <xacro:property name="maximal_axle_steering_angle" value="0.50" />
  <xacro:property name="maximal_wheel_steering_angle" value="1.00" />
  <xacro:property name="maximal_wheel_speed" value="1.0" />


  <xacro:include
    filename="$(find romea_mobile_base_description)/ros2_control/base1FAS2FWD.ros2_control.xacro" />
  <xacro:base_control mode="simulation" base_name="base" />

</robot>